src/SmartComponents/Recs/Details.js

```diff
@@ -184,86 +197,80 @@ const Details = ({
                         <Stack hasGutter>
                             <Card>
                                 <CardBody>
-                                    <ExpandableSection
-                                        toggleText={details.description}
-                                        onToggle={() => setExpanded(!expanded)}
-                                        isExpanded={expanded}
-                                    >
```

Comment:
Do we really intend to remove the expandable section? I can't find it on mocks/mentioned in tickets

Reply:
Yes, as per the requirement expandable section shouldn't be shown in case of EA rules as we will be rendering details.description in Troubleshoot section for them.